docs/installation.md

```diff
@@ -52,15 +52,15 @@ Then you can either pull the container from the remote github repository
 
     .. code-block:: console
 
-        $ apptainer pull docker://ghcr.io/ml4gw/amplfi/data:main $AMPLFI_CONTAINER_ROOT/amplfi.sif 
+        $ apptainer pull $AMPLFI_CONTAINER_ROOT/amplfi.sif docker://ghcr.io/ml4gw/amplfi/data:main
 
     Supported python versions: 3.9-3.12.
 
   .. tab:: docker
 
     .. code-block:: console
 
-      $ docker pull ghcr.io/ml4gw/amplfi:main $AMPLFI_CONTAINER_ROOT/amplfi.sif 
+      $ docker pull ghcr.io/ml4gw/amplfi/data:main 
```

Comment:
The `amplfi` container should be the one stop shop for both datagen and training. I should remove the `data` and `train ` container releases from the repo

Reply:
done